x11-libs/fltk/fltk-1.3~git.recipe

```diff
@@ -13,45 +13,47 @@ HOMEPAGE="https://www.fltk.org/"
 COPYRIGHT="1998-2019 by Bill Spitzak and others."
 LICENSE="FLTK
 	GNU LGPL v2"
-REVISION="1"
-#SOURCE_URI="git+https://github.com/mmuman/fltk.git#haiku-1.3-final"
-SOURCE_URI="https://github.com/mmuman/fltk/archive/haiku-1.3-final.zip"
-#CHECKSUM_SHA256=""
-CHECKSUM_SHA256="a57ed6729dcf1d315ca63912f6660ada1c3548289366ff34bdcfa38b8d45eb89"
-#SOURCE_DIR="qt-everywhere-src-${portVersion/\~/-}"
+REVISION="2"
+SOURCE_URI="https://github.com/mmuman/fltk/archive/haiku-1.3-final.tar.gz"
+CHECKSUM_SHA256="5ea5eb6f4028534bb911a61ad3df7dbfa1274dd200a03be441b75b691f79d7f1"
 SOURCE_DIR="fltk-haiku-1.3-final"
-
-#PATCHES="qt5-$portVersion.patchset"
+PATCHES="fltk-$portVersion.patchset"
 
 ARCHITECTURES="!x86_gcc2 ?x86 ?x86_64"
 SECONDARY_ARCHITECTURES="?x86"
 
+commandSuffix=$secondaryArchSuffix
+commandBinDir=$binDir
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+	commandSuffix=
+	commandBinDir=$prefix/bin
+fi
+
+libVersion="1.3"
+
 PROVIDES="
-	fltk$secondaryArchSuffix = $portVersion compat >= 1.3
-	lib:libfltk$secondaryArchSuffix = $portVersion compat >= 1.3
-	lib:libfltk_forms$secondaryArchSuffix = $portVersion compat >= 1.3
-	lib:libfltk_images$secondaryArchSuffix = $portVersion compat >= 1.3
+	fltk$secondaryArchSuffix = $portVersion
+	lib:libfltk$secondaryArchSuffix = $libVersion
```

Comment:
Not sure here, there is only **libfltk.so.1.3** (**libfltk.so** links to that), so I presume we don't need compat >= ?